src/core/unit.c

```diff
@@ -2704,6 +2704,25 @@ bool unit_can_serialize(Unit *u) {
         return UNIT_VTABLE(u)->serialize && UNIT_VTABLE(u)->deserialize_item;
 }
 
+static int unit_serialize_cgroup_mask(FILE *f, const char *key, CGroupMask mask) {
+        _cleanup_free_ char *s = NULL;
+        int r = 0;
+
+        assert(f);
+        assert(key);
+
+        if (mask) {
```

Comment:
we generally avoid relying on C's downgrade-to-bool feature unless the variable is actually bool-like or a pointer. For numeric values (and this one is one, given it's a bit mask), please compare explicit with != 0 or so. Also see CODING_STYLE.